packages/jest-expect/__typetests__/jest-expect.test.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  expectAssignable,
+  expectError,
+  expectNotAssignable,
+  expectType,
+} from 'tsd-lite';
+import {jestExpect} from '@jest/expect';
+import {expect} from 'expect';
+
+expectType<void>(jestExpect({}).toMatchSnapshot());
+
+expectError(() => {
+  expect({}).toMatchSnapshot();
+});
+
+expectType<void>(jestExpect.addSnapshotSerializer({} as any));
```

Comment:
This is lost after bundling, because of augmentation. I will try `.d.ts` file and `/// <reference path="..." />`

Reply:
huh. might be a bug in the bundler? Feels like `declare module 'foobar'` should never be elided

Reply:
No luck with triple slash. Sounds like this issue: https://github.com/microsoft/rushstack/issues/1709